include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -283,6 +283,10 @@ SECTIONS
 	}
 #endif
 
+#if defined(CONFIG_NOCACHE_MEMORY)
+#include <linker/ram-nocache.ld>
+#endif
```

Comment:
Another thing: this looks like Kernel RAM with no-cache attribution. Should, we , then add this section after APP SMEM and APP MEMORY? Just to group all kernel SRAM together.